apps/settings/js/panels/homescreens_list/update_check.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Home screen update checking functionality.
+ *
+ * @module UpdateCheck
```

Comment:
please remove this file, since it needs rewrite and put into modules/ when the related issue is resolved